[IMP] accounting: journal items menu (#19630)
odoo/documentation: content/applications/finance/accounting/reporting/analytic_accounting.rst

Analytic distribution en masse
------------------------------

To mass-edit analytic accounts in several entries simultaneously, open the **Accounting** app, go to

> :menuselection:`Review --> Journal items`, and select the ones that need to be updated. Click the

Reviewer: It was decided to keep the menuselection like it was before (Accounting > Review > Journal items) to avoid confusion and exception rules (very few apps include a sub-menu named after the app).

```suggestion
To mass-edit analytic accounts in several entries simultaneously, go to :menuselection:`Accounting
--> Accounting --> Journal items`, and select the ones that need to be updated. Click the
```

Author: I changed it back to what it was, but please note the correct path is Accounting --> Review --> Journal Items, not Accounting --> Accounting --> Journal Entries :)

Reviewer: You're right! Sorry about that 🤭

:guilabel:`Analytic Distribution` column and add the required distribution in the
:guilabel:`Analytic` column, then click the :icon:`oi-close` :guilabel:`(cross)` and
:guilabel:`Update`. The analytic distribution is then added to the selected journal items.

.. tip::
   If you do not see the :guilabel:`Analytic Distribution` column, make sure you enabled it in the
   view filter by clicking the :icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon, and

>    ticking :guilabel:`Analytic Distribution`.

Reviewer: I would simplify it to make it more straightforward. What do you think?

```suggestion
   If the :guilabel:`Analytic Distribution` column is not displayed, use the 
   :icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon to add it.
```

------------------------------

To mass-edit analytic accounts in several entries simultaneously, go to :menuselection:`Accounting
--> Review --> Journal items`, and select the ones that need to be updated. Click the
:guilabel:`Analytic Distribution` column and add the required distribution in the
:guilabel:`Analytic` column, then click the :icon:`oi-close` :guilabel:`(cross)` and
:guilabel:`Update`. The analytic distribution is then added to the selected journal items.

.. tip::
   If the :guilabel:`Analytic Distribution` column is not displayed, use the
   :icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon, and ticking

>    :guilabel:`Analytic Distribution`.

Reviewer: ```suggestion
   :icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon, and tick :guilabel:`Analytic
   Distribution`.
```
